dotnet/src/Connectors/Connectors.UnitTests/Memory/Postgres/PostgresMemoryStoreTests.cs

```diff
@@ -0,0 +1,721 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.AI.Embeddings;
+using Microsoft.SemanticKernel.Connectors.Memory.Postgres;
+using Microsoft.SemanticKernel.Memory;
+using Npgsql;
+using Xunit;
+
+namespace SemanticKernel.Connectors.UnitTests.Memory.Postgres;
+
+/// <summary>
+/// Unit tests of <see cref="PostgresMemoryStore"/>.
+/// </summary>
+public class PostgresMemoryStoreTests : IDisposable
```

Comment:
nit: Technically these seem like integration tests, so might be better to put there. Alternatively/additionally you could mock out the `NpgsqlConnection` for unit tests.

Reply:
I moved `PostgresMemoryStoreTests.cs` to the IntegrationTests project. It doesn't seem to mock `NpgsqlConnection`, which is a sealed class. If replace it with `IDbConnection`, there will be no asynchronous support.